virtual_ship/sailship.py

```diff
@@ -1,219 +1,83 @@
 """sailship function."""
 
+from __future__ import annotations
+
 import os
-from datetime import timedelta
+from contextlib import contextmanager
+from dataclasses import dataclass, field
+from datetime import datetime, timedelta
+from typing import Generator
 
-import numpy as np
 import pyproj
 
 from .costs import costs
+from .instrument_type import InstrumentType
 from .instruments.adcp import simulate_adcp
 from .instruments.argo_float import ArgoFloat, simulate_argo_floats
 from .instruments.ctd import CTD, simulate_ctd
 from .instruments.drifter import Drifter, simulate_drifters
 from .instruments.ship_underwater_st import simulate_ship_underwater_st
 from .location import Location
-from .postprocess import postprocess
+from .planning_error import PlanningError
+from .priority_queue import PriorityQueue
 from .spacetime import Spacetime
-from .virtual_ship_configuration import VirtualShipConfig
+from .virtual_ship_config import VirtualShipConfig
+from .waypoint import Waypoint
 
 
 def sailship(config: VirtualShipConfig):
     """
-    Use parcels to simulate the ship, take ctd_instruments and measure ADCP and underwaydata.
+    Use parcels to simulate a cruise.
```

Comment:
We are changing the name of the code to Parcels with capital P in normal text (like NumPy) 
```suggestion
    Use Parcels to simulate a cruise.
```